generate-times.js

```diff
@@ -7,7 +7,7 @@ import vendor1 from './vendor-1.js'
 import { generateSeries } from './common.js'
 
 (async ()=> {
-  const count = 16
+  const count = 8
```

Comment:
8 works for sure, there is a run that finished for 8 in the logs.

I retry with 10 here: https://github.com/OR13/decanonicalization/actions/runs/4064872353